label_sync/repos.yaml

```diff
@@ -0,0 +1,242 @@
+Org: kubernetes
```

Comment:
Why do we need this file? Ideally the labels syncer can just do a github list repos call to sync all repos under the org. That way we don't need to update this file when we add a new repo.